docs/topics/inline-classes.md

```diff
@@ -31,6 +21,10 @@ To declare an inline class for the JVM backend, use the `value` modifier along w
 value class Password(private val s: String)
 ```
 
+> The `inline` modifier is deprecated with the `WARNING` level.
```

Comment:
This can sound scary for those who write inline functions :) Let's be more precise and add "on classes".

I'm also not sure we need the detail about the level. Users will see the warning anyway, nothing will break, but we will need to update this sentence for the next steps of the deprecation cycle.

Reply:
Yes, that may sound intimidating :)

Maybe we should remove information about this altogether? I have checked other features, and I haven't found such a description of the levels.

Reply:
I've updated the description :)